site/get-data.js

```diff
@@ -1,5 +1,7 @@
 const swagImages = [];
 const fileNames = [];
+
+delete require.cache[require.resolve('../data.json')];
```

Comment:
If we're following the dev suggestion of not using `gulp watch` with changes to the `data.json` file, this isn't necessary